static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -19,24 +19,24 @@ import fastdom from 'fastdom';
 import raven from 'lib/raven';
 import userPrefs from 'common/modules/user-prefs';
 import images from 'common/modules/ui/images';
-import storage from 'lib/storage';
+import { localStorage } from 'lib/storage';
```

Comment:
hmm that's a very a good point. yeah i guess it will be scoped but it's ripe for confusion and error... 

`storageLocal`?